keps/sig-windows/1981-windows-privileged-container-support/README.md

```diff
@@ -249,10 +252,10 @@ What are some important details that didn't come across above?
 Go in to as much detail as necessary here.
 This might be a good place to talk about core concepts and how they relate.
 -->
+
 - Host network mode support is only targeted for privileged containers and pods.
 - Privileged pods can only consist of privileged containers. Standard Windows Server containers or other non-privileged containers will not be supported. This is because containers in a Kubernetes pod share an IP. For the privileged containers with host network mode, this container IP will be the host IP. As a result, a pod cannot consist of a privileged container with the host IP and an unprivileged Windows Server container(s) sharing a vNic on the host with a different IP, or vice versa.
-- We are currently investigating service mesh scenarios where privileged containers in a pod will need host networking access but run alongside non-privileged containers in a pod. This would require further changes and investigation.
-
+- We are currently investigating service mesh scenarios where privileged containers in a pod will need host networking access but run alongside non-privileged containers in a pod. This would require further investigation and is out of scope for this KEP.
```

Comment:
If this is a possibility in the future, should we keep `privileged` at the container level in the Kubernetes API (keep the API the same with linux)?

If all containers need to be privileged now, we can add API validation for it.

The currently proposed Kubernetes API change is a bit confusion to me.

Reply:
@dcantah @kevpar were still doing some investigations here.
Do we think it is possible to ever allow of mix of privileged and non-privileged containers in the same pod?

Also @Random-Liu if we follow my proposal above (https://github.com/kubernetes/enhancements/pull/2288#discussion_r566439189) we wouldn't need to specify a pod-wide privileged field. Maybe that is the best route to take. 

Thoughts?

Reply:
Yes it should be possible, just needs some OS work (on the networking side) to happen which is always a pain. I don't think the HNS folks have started on this yet